### DISCUSSION TITLE
[Q & A] How Can I install programming language runtimes like node.js, python3 and jdk

### DISCUSSION THREAD

**Discussion by gyunseo** [aquaproj/aqua#2574] [Deprecated (Question and Answer)] [answered]
### Question

I'm trying to manage runtime versions using `aqua`.
So I installed it on my Windows machine reading official docs.
However, when i use `aqua g` command, there is only `bun` runtime. 
I want to use another programming language runtimes.
Could you please let me know how to do it?

Best Regards
@gyunseo

### Example Code

```console
$ 
```

Configuration

```yaml

```

**Comment by gyunseo** [comment]
- <https://zenn.dev/shunsuke_suzuki/books/aqua-handbook/viewer/what-aqua#aqua-%E3%81%AF%E5%85%A8%E3%81%A6%E3%82%92%E4%BB%A3%E6%9B%BF%E3%81%A7%E3%81%8D%E3%82%8B%E3%81%AE%E3%81%8B>

> 特に asdf がサポートしている Python や Ruby, Node.js などは aqua はサポートしていません。   

I've just read the post above! I've just realized there is no support for  Python, Ruby, Node.js runtime.

**Reply by suzuki-shunsuke** [reply, marked as answer]
Correct. Regarding programming language runtimes, aqua supports runtimes such as Go, Deno, and Bun, but doesn't support Python, Ruby, and Node.js.
aqua has several limitations so it's difficult to support these runtimes.

**Reply by suzuki-shunsuke** [reply]
Update:

aqua has supported Node.js 🎉

https://aquaproj.github.io/docs/reference/nodejs-support/

**Reply by neitsab** [reply]
Would you consider adding a "supported tools" section to the README? I ended up here after looking for which tools aqua supported for several minutes, and it's pretty important to know whether it fits one's use case or not 😁
Thanks!

**Reply by suzuki-shunsuke** [reply]
1. You can search tools `aqua g` command.

https://aquaproj.github.io/docs/tutorial/search-packages

```
aqua g
```

2. You can also output supported tools by `aqua list` command.

```
aqua list
```

3. You can search tools in https://github.com/aquaproj/aqua-registry

e.g.

<img width="752" alt="image" src="https://github.com/user-attachments/assets/e8a77d5b-9a67-4d63-8a0a-88c287efe6ef">

--

<img width="836" alt="image" src="https://github.com/user-attachments/assets/dc2989f0-0343-4a16-a3cf-9235433ab42f">

```console
$ git clone https://github.com/aquaproj/aqua-registry
$ cd aqua-registry
$ git grep -l tfenv
pkgs/tfutils/tfenv/pkg.yaml
pkgs/tfutils/tfenv/registry.yaml
registry.yaml
```

**Reply by suzuki-shunsuke** [reply]
> it's pretty important to know whether it fits one's use case or not

That's a good point.
Maybe we should add the link to the guide in https://aquaproj.github.io/ or https://aquaproj.github.io/docs/
